packages/frontapp/src/app/pages/HomePage.tsx

```diff
@@ -181,28 +181,23 @@ const HomePage: FC = () => {
           )}
         </>
       ) : (
-        <>
-          <ButtonWrapper>
-            <Button value={t(`ManagerInfoCard.Description.Library`)} />
```

Comment:
The button should be here, but currently there is no correspond design for mobile. We could leave it until Ekaterina will provide us new design

Reply:
should i make it as a comment?

Reply:
Yep, you should not remove that button. leave a comment